first_hour_on_vwb/working_with_resources.ipynb

```diff
@@ -0,0 +1,1436 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "'''\n",
+    "Resolves bucket URL from bucket reference in workspace.\n",
+    "'''\n",
+    "def get_bucket_url_from_reference(bucket_reference):\n",
+    "    BUCKET_CMD_OUTPUT = !terra resolve --name={bucket_reference}\n",
+    "    BUCKET = BUCKET_CMD_OUTPUT[0]\n",
+    "    return BUCKET\n",
+    "\n",
+    "'''\n",
+    "Resolves current workspace ID from workspace description.\n",
+    "'''\n",
+    "def get_current_workspace_id():\n",
+    "    WORKSPACE_CMD_OUTPUT = !terra workspace describe --format=json | jq --raw-output \".id\"\n",
+    "    WORKSPACE_ID = WORKSPACE_CMD_OUTPUT[0]\n",
+    "    return WORKSPACE_ID\n",
+    "\n",
+    "CURRENT_WORKSPACE_ID = get_current_workspace_id()\n",
+    "print(f'Workspace ID: {CURRENT_WORKSPACE_ID}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### Workspace setup\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Note:</b> This notebook assumes that <a href=\"../../terra-axon-examples/workspace_setup.ipynb\">`workspace_setup.ipynb`</a> has been run.\n",
+    "</div>\n",
+    "    \n",
+    "`workspace_setup.ipynb` creates two Cloud Storage buckets for your workspace files with workspace reference names:\n",
+    "\n",
+    "- ws_files\n",
+    "- ws_files_autodelete_after_two_weeks\n",
+    "\n",
+    "The code in this notebook will write output files to the \"autodelete\" bucket by default.  \n",
+    " Any file in this bucket will be automatically deleted <b>two weeks</b> after it is written.  \n",
+    " This alleviates the need for you to remember to clean up temporary and example files manually.  \n",
+    " If you want to write outputs to a durable location, simply change the assignment of the `BUCKET_REFERENCE` variable in the cell below and re-run the notebook.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Change this to \"ws_files\" to use the durable workspace bucket instead of the autodelete bucket.\n",
+    "BUCKET_REFERENCE = \"ws_files_autodelete_after_two_weeks\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "MY_BUCKET = get_bucket_url_from_reference(BUCKET_REFERENCE)\n",
+    "print(f'Bucket ID: {MY_BUCKET}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Add referenced resources\n",
+    "\n",
+    "[Referenced resources](https://terra-docs.api.verily.com/docs/getting_started/web_ui/#referenced-vs-workspace-controlled-resources) represent data and other elements in Enterprise Terra by pointing to a source that exists outside of the current workspace.\n",
```

Comment:
There's a few stray references to Enterprise Terra that should be VWB in this section

Reply:
fixed